cpp/src/slice2swift/SwiftUtil.cpp

```diff
@@ -1612,7 +1612,9 @@ SwiftGenerator::writeDispatchOperation(::IceInternal::Output& out, const Operati
     out << sb;
     out << nl;
 
-    // TODO: check operation mode
+    if (op->mode() == Operation::Mode::Normal) {
```

Comment:
This is actually the most correct check, and what we do already in IceRPC:
https://github.com/icerpc/icerpc-csharp/blob/f109b98c1cb69755ccca7ddf3cc247982e7afb8c/src/IceRpc.Slice/IncomingRequestExtensions.cs#L19

I'll update the other languages to perform the same check.